newrelic-agent/src/test/resources/com/newrelic/agent/uncross_agent_tests/request_uri_java_only.json

```diff
@@ -141,7 +141,7 @@
     },
     "expectation": {
       "request_uri": [
```

Comment:
This file doesn't actually exist in the shared [cross_agent_tests](https://source.datanerd.us/agents/cross_agent_tests) repo so I moved it into the Java specific `uncross_agent_tests` directory.